### PR TITLE
7.54.0 unfreeze

### PR DESCRIPTION
**Pull request by kacper-murzyn**
Updates after 7.54.x release branch creation.

### CODE REVIEW

File: .gitlab-ci.yml
Comment (chouetz) [inline .gitlab-ci.yml:1]: There is now an occurrence of `compare_to` also in the `.gitlab/notify/notify.yml` file, should we modify it as well?

**Review by alopezz — APPROVED** [review]

File: .gitlab-ci.yml
Comment (kacper-murzyn) [reply .gitlab-ci.yml:1]: Good question, I don't know the answer. Who own those files so we can check?

**Review by L3n41c — APPROVED** [review]

File: .gitlab-ci.yml
Comment (paulcacheux) [reply .gitlab-ci.yml:1]: The comparison is done against main, so yes it definitely needs to be changed

**Review by paulcacheux — APPROVED** [review]

File: .gitlab-ci.yml
Comment (kacper-murzyn) [reply .gitlab-ci.yml:1]: Change added

**Pull request merged**